ncache-vm-cluster/README.md

```diff
@@ -0,0 +1,115 @@
+# NCache Manged ARM Template
```

Comment:
need to add the badges and buttons, see the pipeline task for a snippet (or the contribution guide)